### ISSUE TITLE
Scanpy or Seurat interface

### ISSUE THREAD

**Issue by qinqian** [jmbreda/Sanity#4]
Dear authors, 

Thank you for presenting such a great tool, do you consider implementing the interfaces for common tools like Scanpy or Seurat?

Best,
Qian

**Comment by dominikburri** [comment]
Agree, it would be nice to include the method in Bioconductor to directly get normalised counts from a *SingleCellExperiment* or *Seurat* object.

**Comment by TeoSakel** [comment]
Shameless plug/necropost 😬: https://bioconductor.org/packages/SanityR/
